cloud.go

```diff
@@ -127,7 +127,7 @@ func (cm *CloudManager) DoesRemoteDirExist(storage Storage, remotePath string) (
 	cmd := makeCommand(getCloudApp(), "lsjon", path+"/")
 	err := cmd.Run()
 	if err != nil {
-		return false, nil
```

Comment:
This is an unrelated bug that would cause errors in remote dir creation to not bubble up.

Reply:
On second look, I think we want original behavior here.